### DISCUSSION TITLE
W7Fix - The Shameless Plug

### DISCUSSION THREAD

**Discussion by Burgerballs** [FunkinCrew/Funkin#2110] [Show and tell]
https://github.com/Burgerballs/FNF-W7Fix

This is an engine that is meant to add to the pre-existing Week 7 engine while keeping the original feel, and making all diversions as optional as possible, the engine is at an extremely early stage right now but hey, it's still somethin! The main changes as of right now is the options having nice little descriptions, and the addition of ghost tapping (optional and disabled by default).

**Comment by saragannett** [comment]
i cant find an unblocked version of fnf do you have any websites for chromebook that i can use?
or any proxies
